### DISCUSSION TITLE
Upgrade to use JDK 11 and  Selenide 7.0.3

### DISCUSSION THREAD

**Discussion by rajukthomas** [Xceptance/neodymium-library#233] [Q&A]
It is possible to upgrade the neodymium library to use JDL 11 and the latest version of Selenide. If yes, what are the steps?

**Comment by wurzelkuchen** [comment]
Updating the Selenide version is unfortunately not an easy task. 
We are currently working on this and hopefully will release a new version soon.

**Comment by wurzelkuchen** [comment]
Neo 5.0.0. was just released using Selenide 7 and JDK 17 (which is required by Selenide >7). Hope this helps :slightly_smiling_face:
